bot.py

```diff
@@ -13,7 +13,7 @@
 import util.invites
 from util.log import Log
 from util.db import DbGuild, DbInvite, DbUser, DbCategory, Base
-
+import googletrans
```

Comment:
When adding an external dependency to the bot, make sure to, in your terminal, run `pip freeze > requirements.txt` in order to add this dependency and any of its dependencies to the requirements for the bot. This way, when the bot is pushed to its docker container and an image is built, it will be able to instantly update dependencies.